specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json

```diff
@@ -2469,12 +2469,25 @@
         "typeProperties"
       ]
     },
+    "WebActivityAuthenticationType": {
+      "description": "The list of Authentication types supported by a Web Activity.",
+      "type": "string",
+      "enum": [
+        "Basic",
+        "ClientCertificate",
+        "MSI"
+      ],
+      "x-ms-enum": {
+        "name": "WebActivityAuthenticationType",
+        "modelAsString": true
+      }
+    },
     "WebActivityAuthentication": {
       "description": "Web activity authentication properties.",
       "properties": {
         "type": {
           "description": "Web activity authentication (Basic/ClientCertificate/MSI)",
-          "type": "string"
+          "$ref": "#/definitions/WebActivityAuthenticationType"
```

Comment:
@jhendrixMSFT, Is this considered a breaking change? The new type is an enum that is modeled as a string so in SDK it should be still string.

Reply:
Changing it back to old format